frontend/src/hooks/api/actions/useServiceAccountsApi/useServiceAccountsApi.ts

```diff
@@ -0,0 +1,66 @@
+import useAPI from '../useApi/useApi';
+
+export interface IServiceAccountPayload {
+    name: string;
+    username: string;
+    rootRole: number;
```

Comment:
Not sure what the `rootRole` is, just asking to learn about it.

Reply:
`rootRole` should map to one of our root roles in Unleash: 
 - Admin;
 - Editor;
 - Viewer;
 
 Just like regular users, service accounts also have a root role associated with them: 
<img width="757" alt="image" src="https://user-images.githubusercontent.com/14320932/210565815-0620a08f-e4c1-4c7a-8c2f-7a7592c6d51d.png">